pkg/smb/smb_test.go

```diff
@@ -43,6 +43,10 @@ func TestNewFakeDriver(t *testing.T) {
 }
 
 func TestIsCorruptedDir(t *testing.T) {
+	if isWindows() {
```

Comment:
what about write like this:
```
skipIfTestingInWindows(t)
```

and in func `skipIfTestingInWindows`:
```
	if runtime == windows {
		t.Skip("Skipping tests on Windows")
	}
```

that could reduce code

Reply:
Makes sense! On it :)